testbench/common.py

```diff
@@ -46,6 +47,10 @@
 retry_return_broken_stream_after_bytes = re.compile(
     r"return-broken-stream-after-([0-9]+)K$"
 )
+retry_stall_beginning = re.compile(
+    r"stall-at-beginning-([0-9]+)K$"
```

Comment:
This doesn't say how long to stall for; that information can help to be provided in the instruction as well.